clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -56,6 +57,8 @@ export const ConsentButtons = ({
       onManagePreferencesClick();
     }
   };
+  const includeBrandLink =
+    isInModal && options.showFidesBrandLink && options.fidesBrandUrl;
```

Comment:
nit: Can this just be a single config `fidesShowBrandingLink`, and just set the brand URL to ttps://fid.es/powered instead of making that a config option? (I just created that shortlink now)

Having two config options just makes this more complicated to document & support, and I'd rather have one clear boolean `FIDES_PRIVACY_CENTER__SHOW_BRANDING_LINK` that is an on/off switch.

Reply:
@NevilleS What about a default URL and an optional 2nd config? Or nah, just axe the 2nd config?

Reply:
It's my **Strong Opinion**™️ (but weakly held) that we shouldn't add two config options for this feature unless we have a really compelling reason. I'd question the utility of the URL config- since the branding logo itself isn't configurable in the same way, why would I change it to anything else? Instead it just creates possibilities for invalid configurations where a client doesn't realize the two options are dependent on each other.